tests/math/test_functions.py

```diff
@@ -419,6 +434,8 @@ def test_vector_product(self, t1, t2):
         [tf.Variable([[1, 2], [3, 4]]), onp.array([6, 7])],
         [tf.constant([[1, 2], [3, 4]]), onp.array([6, 7])],
         [tf.Variable([[1, 2], [3, 4]]), tf.Variable([6, 7])],
+        [jnp.array([[1, 2], [3, 4]]), jnp.array([6, 7])],

```

Comment:
Awesome :+1:

Reply:
Is it worth adding a test case that is a combination of vanilla numpy and JAX? e.g., 
```suggestion
        [jnp.array([[1, 2], [3, 4]]), jnp.array([6, 7])],
        [np.array([[1, 2], [3, 4]]), jnp.array([6, 7])],
```
This is something that might arise, e.g., if a template developer uses vanilla numpy internally and the user using the templates passes jax numpy arrays.